### PR TITLE
Soc 624 Localizing social links and hubs for new HTML emails

### PR DESCRIPTION
**Pull request by lizlux**
Previously, the hubs names and social networks were hard-coded in english, as were their urls. This PR i18n's everything.

### CODE REVIEW

File: extensions/wikia/Oasis/Oasis.i18n.php [hunk 1]
```diff
@@ -161,12 +161,19 @@
 your email preferences here: http://community.wikia.com/Special:Preferences',
 	'oasis-label-wiki-vertical-id-0' => 'Other',
 	'oasis-label-wiki-vertical-id-1' => 'TV',
-	'oasis-label-wiki-vertical-id-2' => 'Video Games',
```
Comment (jsutterfield) [inline extensions/wikia/Oasis/Oasis.i18n.php]: Did we decide to change this hub category to "Games"? If so, it looks like all the other translations for this message are translating the term "Video Games", those should be changed as well.

Comment (lizlux) [inline extensions/wikia/Oasis/Oasis.i18n.php]: We're going to leave it as is for now and chat with Noelle about possibly updating other languages at a later date.

File: extensions/wikia/Email/EmailController.class.php [hunk 2]
```diff
@@ -293,6 +291,41 @@ protected function getTagline() {
 	}
 
 	/**
+	 * Get localized strings for hubs names and linkk
```
Comment (jsutterfield) [inline extensions/wikia/Email/EmailController.class.php]: typo in "linkk"

File: extensions/wikia/Email/EmailController.class.php [hunk 3]
```diff
@@ -293,6 +291,41 @@ protected function getTagline() {
...
+			'lifestyle' => wfMessage( 'oasis-label-wiki-vertical-id-5' )->inLanguage( $this->targetLang )->text(),
+			'lifestyleURL' => wfMessage( 'oasis-label-wiki-vertical-id-5-link' )->inLanguage( $this->targetLang )->text(),
+			'music' => wfMessage( 'oasis-label-wiki-vertical-id-6' )->inLanguage( $this->targetLang )->text(),
+			'musicURL' => wfMessage( 'oasis-label-wiki-vertical-id-6-link' )->inLanguage( $this->targetLang )->text(),
+			'movies' => wfMessage( 'oasis-label-wiki-vertical-id-7' )->inLanguage( $this->targetLang )->text(),
+			'moviesURL' => wfMessage( 'oasis-label-wiki-vertical-id-7-link' )->inLanguage( $this->targetLang )->text(),
+		];
+	}
+
+	protected function getSocialMessages() {
```
Comment (jsutterfield) [inline extensions/wikia/Email/EmailController.class.php]: Can you add a docs for this one as well?

**Comment by jsutterfield** [comment]
LGTM

**Pull request merged**